bin/bios-baseline.sh

```diff
@@ -70,7 +93,9 @@ function ilo_config() {
 }
 
 function ilo_enable_ipmitool {
```

Comment:
So the function name is `enable_impitool`.  Maybe the option name should be changed instead? `--check-compliance`?  `--verify`?  `--report`?

Reply:
I feel like all of those option names still give the impression that no changes would be made.

During installs (which AFAIK is the only time this script is run, since it's on the PIT node), the usual sequence is that this script is run WITHOUT the --check flag first. So any changes needed (including enabling ipmitool) will be made at that point. The script is later called with the --check argument as one of the preflight checks. And just like the names of the flags, I think most people don't expect the tests to make configuration changes either.

IMO it is fine to not try to enable ipmitool at that point. If they do want to enable it, they can always re-run the script without the --check argument.

If the concern is merely that the function name says it is enabling ipmitool, but we don't end up doing it, it's easy enough to implement another way -- add logic where we only call the function is --check is not specified. I actually had it coded that way originally but this way seemed cleaner. But I don't feel strongly about it. I will change it to the other way.

Reply:
Ok, I have made the change and tested it on surtur.

Reply:
Specifically, @heemstra , does my change allay your concerns? I do not wish to incur your terrible wrath!!

Reply:
Yep, I dig it.